enterprise-beans/4.0/_index.md

```diff
@@ -1,23 +1,28 @@
 ---
-title: "Jakarta Enterprise Beans 4.0 (milestone)"
-date: 2020-06-05
+title: "Jakarta Enterprise Beans 4.0"
+date: 2020-08-26
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
 * [Jakarta Enterprise Beans 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan)
   * [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-ee4j/ejb-api/blob/master/4.0-PLAN.adoc)
-* [Jakarta Enterprise Beans, Core Features 4.0 Specification Document](./enterprise-beans-spec-core-4.0-public-draft.pdf) (PDF)
-* [Jakarta Enterprise Beans, Core Features 4.0 Specification Document](./enterprise-beans-spec-core-4.0-public-draft.html) (HTML)
-* [Jakarta Enterprise Beans, Optional Features 4.0 Specification Document](./enterprise-beans-spec-opt-4.0-public-draft.pdf) (PDF)
-* [Jakarta Enterprise Beans, Optional Features 4.0 Specification Document](./enterprise-beans-spec-opt-4.0-public-draft.html) (HTML)
-* [Jakarta Enterprise Beans 4.0.0-RC2 Javadoc](./apidocs)
-* [Jakarta Enterprise Beans 4.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Enterprise Beans, Core Features 4.0 Specification Document](./jakarta-enterprise-beans-spec-core-4.0.pdf) (PDF)
+* [Jakarta Enterprise Beans, Core Features 4.0 Specification Document](./jakarta-enterprise-beans-spec-core-4.0.html) (HTML)
+* [Jakarta Enterprise Beans, Optional Features 4.0 Specification Document](./jakarta-enterprise-beans-spec-opt-4.0.pdf) (PDF)
+* [Jakarta Enterprise Beans, Optional Features 4.0 Specification Document](./jakarta-enterprise-beans-spec-opt-4.0.html) (HTML)
+* [Jakarta Enterprise Beans 4.0 Javadoc](./apidocs)
+* [Jakarta Enterprise Beans 4.0 TCK](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/jakarta-jakartaeetck-9.0.0-RC1.zip)
```

Comment:
These links should be the final resting location of the TCK:  https://download.eclipse.org/jakartaee/platform/9/jakarta-jakartaeetck-9.0.0.zip (just like you have for the sig and sha files).

Reply:
Done